testing/tests/DevExpress.ui.widgets/popover.tests.js

```diff
@@ -2284,3 +2284,27 @@ QUnit.module('target option', {
     });
 });
 
+QUnit.module('aria accessibility', {
+    beforeEach: function() {
+        fixtures.simple.create();
+    },
+    afterEach: function() {
+        fixtures.simple.drop();
+    }
+}, () => {
+    QUnit.test('role="tooltip" attribute should be added to popover by default', function(assert) {
+        const $popover = $('#what');
+            new Popover($popover);
+            const $overlay = $('.dx-overlay-content');
+
+            assert.equal($overlay.attr('role'), 'tooltip');
+    });
+   
+    QUnit.test('role="dialog" attribute should be set if popover has toolbarItems', function(assert) {
+        const $popover = $('#what');
+            new Popover($popover, { toolbarItems: [{ text: "Title", location: "before" }]});
```

Comment:
seems like linter does not work for your unit tests.
Please, check it and fix all miss formating